rotkehlchen/chain/ethereum/abi.py

```diff
@@ -57,7 +57,7 @@ def decode_event_data_abi(
     May raise:
     - DeserializationError if the abi string is invalid or abi or log topics/data do not match
     """
-    if event_abi['anonymous']:
+    if event_abi.get('anonymous', False):
```

Comment:
This also I am not totally sure about ... it's marked as NotRequired. But it seems to always be there in the typedict of web3